.github/workflows/prettier.yml

```
@@ -0,0 +1,27 @@
+name: Prettier
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+    branches: 
+      - main
```

I don't think this needs to be so explicit about the `branches` and `types` for the `pull_request` event? Using `pull_request:` without any children should include everything.

```suggestion
on:
  push:
    branches:
      - main
  pull_request:
```

[Example from the TypeStat repo](https://github.com/JoshuaKGoldberg/TypeStat/blob/3adf2e0ded053e9f858820386a43f3142258bbd9/.github/workflows/push.yml#L26)